flow/aggregates.py

```diff
@@ -36,15 +66,18 @@ def foo(*jobs):
         A callable that performs aggregation of jobs. It takes in a list of
         jobs and can return or yield subsets of jobs as an iterable. The
         default behavior is creating a single aggregate of all jobs.
-    sort_by : str or None
-        Before aggregating, sort the jobs by a given statepoint parameter.
-        The default behavior is no sorting.
+    sort_by : str, callable, or None
+        Before aggregating, sort the jobs by a given statepoint parameter. If
+        the argument is callable, this will be passed as the callable
+        argument to :func:`sorted`. If None, no sorted is performed. (Default
+        value = None)
```

Comment:
I don't think we're inconsistent - I think we use the period-before in most/all cases. That style was favored at some point by a formatting tool, perhaps when converting from Sphinx-style to NumPy-style docstrings. I just stuck with it.

Reply:
I checked through signac, signac-flow, freud, and coxeter as a set of representative examples (just a simple ripgrep for "Default value"). While we're not 100% consistent anywhere, signac, freud, and coxeter all have a majority (I'd say ~80%) of the period after. The only exception is signac-flow, and since you did a lot of work on the numpydoc conversion as well as your recent dev pushes I suspect that you have (consistently) chosen the opposite convention we've preferred in all our other code and probably changed preexisting period-after docstrings. Let's revert those changes incrementally as we find them.